src/CSnakes.Runtime/ServiceCollectionExtensions.cs

```diff
@@ -179,6 +179,18 @@ public static IPythonEnvironmentBuilder FromConda(this IPythonEnvironmentBuilder
         return builder;
     }
 
+    public static IPythonEnvironmentBuilder FromManagedPython(this IPythonEnvironmentBuilder builder)
```

Comment:
Not wedded to this name. Suggestions welcome

Reply:
`FromPythonDownload`?

`FromPythonRedistributable`?

Reply:
`FromRedistributable` makes sense since none of the others have Python in the name